notebooks/SITCOMTN_131_M1M3_VMS_TMA_Coupling_investigation_strong_shaking-28-06-2023.ipynb

```diff
@@ -0,0 +1,840 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    vms_dir = os.path.join("/scratch/users/b/boutigny/vmsdata", year, month)
</pre><p>Substitute explicit path with vms_top_dir. Is there some generic place we can have this data that does not rely on your scratch space?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/sitcomtn-131/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Unfortunately not. Petr was supposed to move these files to the LFA but he is way too busy to do it.</p>